PULL_REQUEST_TEMPLATE

```diff
@@ -21,11 +21,4 @@ Add a description of your changes here.
 
 ### Reviewers
 
-If you've been tagged for review, please "Start a review" with the GitHub GUI.
-
-- Code review: @Reviewer1, @Reviewer2
-- UI strings/error msgs review: @...
-- UX review: @...
-- Accessibility review: @...
-- Product review: @...
-- FYI @someone
+Add reviewers with the github GUI. 
```

Comment:
External contributors would go through the standard OSPR process. Expecting them to request reviews from (or even have knowledge of) the appropriate edX members themselves doesn't seem reasonable.